packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -23,6 +22,7 @@ interface Props {
     className?: string;
     helpLabel?: string;
     errorLabel?: string;
+    variant?: "secondary" | "small";
```

Comment:
We might want to move this type to `core` to make it easier to sync with eventual changes to label variants